README.md

```diff
@@ -1,24 +1,72 @@
-# Apache Hive docs site
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
 
-This directory contains the code for the Apache Hive web site,
-[hive.apache.org](https://hive.apache.org/). The easiest way to build
-the site is to use docker to use a standard environment.
+  http://www.apache.org/licenses/LICENSE-2.0
 
-## Run the docker container with the preview of the site.
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License. -->
+# Apache Hive Documentation Site
 
-1. `docker build -t hive-site .`
-2. `CONTAINER=$(docker run -d -p 4000:4000 hive-site)`
+This repository contains the code for generating the Apache Hive web site.
+It's built with Hugo and hosted at https://hive.apache.org.
 
-## Browsing
+### Build and Run Locally
 
-Look at the site by navigating to
-[http://0.0.0.0:4000/](http://0.0.0.0:4000/) .
+* Clone this repository.
+* Install [hugo]:
 
-## Pushing to site
+ ```brew install hugo```
```

Comment:
Should we mention the installation procedure for different OS like windows/linux?

Reply:
I have only tested out in MacOS. 
For other OS I have added a link to hugo installation page https://gohugo.io/installation/